proposals/0031-hlsl-vector-matrix-operations.md

```diff
@@ -665,6 +667,11 @@ void OuterProductAccumulate(
 } // namespace dx
 ```
 
+Validation:
```

Comment:
If we are talking about HLSL diagnostics, we should say "Diagnostics" not "Validation".  We use "Validation" to refer to DXIL Validation.